Automattic/wp-calypso
# Stats: Releasing Devices module.

Related to #

## Proposed Changes

* release the Devices module

## Testing Instructions



* open the live branch
* test various types of blogs - internal, external, simple, atomic, with and without Jetpack
* make sure the module works properly for the correct subscription type

## Pre-merge Checklist
